src/protocol/products/active_products_response.rs

```diff
@@ -13,3 +16,19 @@ pub struct ActiveProduct {
     pub name: String,
     pub price: String,
 }
+
+// TODO: Should not be here
```

Comment:
Fix todo now or later?

Reply:
This is gonna be moved when it is used in more than one place.